pyuvdata/uvbeam/uvbeam.py

```diff
@@ -1447,6 +1447,12 @@ def _interp_healpix_bilinear(
         if self.pixel_coordinate_system != "healpix":
             raise ValueError('pixel_coordinate_system must be "healpix"')
 
+        if not self.Npixels == 12 * self.nside ** 2:
+            raise ValueError(
+                "simple healpix interpolation requires full sky healpix maps."
+            )
+        assert np.max(np.abs(np.diff(self.pixel_array))) == 1
```

Comment:
I'm typically only a fan of bare `assert` statements if this would be something set internally, and not touched by the user. If that's the case here, we can keep it (it's not clear from this snippet if it is or not...).

Reply:
This can only happen if the pixels are re-ordered on the object, which can't easily be done by users. I can make it a test with a value error if you prefer, I moved this from inside a loop to make it fail faster.

Reply:
I fixed it.